source/features/clean-issue-filters.tsx

```diff
@@ -0,0 +1,48 @@
+import select from 'select-dom';
+import features from '../libs/features';
+import * as api from '../libs/api';
+import {getOwnerAndRepo} from '../libs/utils';
+
+async function init(): Promise<void> {
+	const {ownerName, repoName} = getOwnerAndRepo();
+	const result = await api.v4(`
+		query {
+			repository(owner: "${ownerName}", name: "${repoName}") {
+				projects { totalCount }
+				milestones { totalCount }
+			},
+			organization(login: "${ownerName}") {
+				projects { totalCount }
+			}
+			user(login: "${ownerName}") {
+				projects { totalCount }
+			}
```

Comment:
Apparently even if I have a personal project, public, with an issue from the current repo inside, the dropdown is still empty for me.

I dropped this